<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
        background-color: #f0f2f5;
        font-family: 'Segoe UI', 'Arial', sans-serif;
        font-size: 14px;
        color: #333;
    }

    QLabel#titleLabel {
        font-size: 24px;
        font-weight: bold;
        color: #0078d4;
        padding-bottom: 10px;
        qproperty-alignment: AlignHCenter;
    }

    QLineEdit#cityInput {
        padding: 6px 10px;
        border: 1px solid #ccc;
        border-radius: 5px;
        background-color: #fff;
        min-width: 200px;
    }

    QPushButton#fetchButton {
        background-color: #0078d4;
        color: white;
        border: none;
        padding: 6px 15px;
        border-radius: 5px;
    }

    QPushButton#fetchButton:hover {
        background-color: #005a9e;
    }

    QLabel#weatherLabel,
    QLabel#feelsLikeLabel,
    QLabel#humidityLabel,
    QLabel#windLabel {
        font-size: 16px;
        padding: 4px;
    }

    QLabel#iconLabel {
        margin-top: 10px;
        qproperty-alignment: AlignHCenter;
    }</string>
  </property>
  <widget class="QWidget" name="backgroundFrame">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Weather App</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="cityInput">
         <property name="placeholderText">
          <string>Enter city name...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fetchButton">
         <property name="text">
          <string>Get Weather</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="iconLabel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="weatherLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="feelsLikeLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="humidityLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="windLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
